### PR TITLE
feat(raw_vehicle_cmd_converter): use polling subscriber

### PR DESCRIPTION
**Pull request by zusizusi**
## Description

The same as https://github.com/autowarefoundation/autoware.universe/pull/6997 based on [the discussion](https://github.com/orgs/autowarefoundation/discussions/4612), the polling subscriber is used in the raw_vehicle_cmd_converter.

## Tests performed

Since there was no ROSbag file for the new topic type, I changed the topic type in the code and verified that the topic was output using the ROSbag file of the old topic type.

## Effects on system behavior

Nothing but more efficient CPU usage

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: vehicle/raw_vehicle_cmd_converter/src/node.cpp [hunk 1]
```diff
@@ -204,20 +200,30 @@ double RawVehicleCommandConverterNode::calculateBrakeMap(
...
-void RawVehicleCommandConverterNode::onVelocity(const Odometry::ConstSharedPtr msg)
+void RawVehicleCommandConverterNode::processOdometry(const Odometry::ConstSharedPtr msg)
 {
+  if (!msg) {
+    return;
+  }
   current_twist_ptr_ = std::make_unique<TwistStamped>();
   current_twist_ptr_->header = msg->header;
   current_twist_ptr_->twist = msg->twist.twist;
 }
```
Comment (shmpwk) [inline vehicle/raw_vehicle_cmd_converter/src/node.cpp:219]: IMO
They are no longer a callback, but simply an assignment operation, I prefer to sum up in `onControlCmd`.
Actually, `onControlCmd` can be replaced to timer callback, but this time I believe that is out of this PR and look forward you to the future contribution.

**Comment by shmpwk** [comment]
@zusizusi 
Thank you for the contribution. What about being the code owner like this?
https://github.com/autowarefoundation/autoware.universe/blob/0b4969a4fe7197162ed4aed19723a0f5014561ac/vehicle/external_cmd_converter/package.xml#L8

File: vehicle/raw_vehicle_cmd_converter/src/node.cpp (hunk 1 above)
Comment (zusizusi) [reply vehicle/raw_vehicle_cmd_converter/src/node.cpp]: Thank you for your review. I sum up the functions.

**Review by shmpwk — APPROVED** [review]
LGTM!

**Pull request merged**